File: Notification System/src/MCPNotifier.cpp
#include "MCPNotifier.hpp"
#include <iostream>

void MCPNotifier::sendNotification(const std::string& sensorId,
                                   const std::string& message,
                                   const std::string& timestamp) {
    std::cout << "[MCP/LLM] " << timestamp
              << " | Capteur: " << sensorId
              << " | Message: " << message << std::endl;
    // TODO: implémenter appel à LLM ou MCP pour traitement
}

File: Notification System/notifiers/mcp_notifier.py
from .notifier import Notifier


class MCPNotifier(Notifier):

    def send_notification(self, sensor_id: str, message: str, timestamp: str):
        print(f"[MCP/LLM] {timestamp} | Capteur: {sensor_id} | Message: {message}")
        # TODO: implémenter appel à LLM ou MCP
